torchvision/datasets/folder.py

```diff
@@ -203,7 +203,7 @@ def make_dataset(
         return make_dataset(directory, class_to_idx, extensions=extensions, is_valid_file=is_valid_file)
 
     @staticmethod
-    def _find_classes(dir: str) -> Tuple[List[str], Dict[str, int]]:
+    def find_classes(dir: str) -> Tuple[List[str], Dict[str, int]]:
```

Comment:
Let's make this an instance method too and remove the `@staticmethod` following https://github.com/pytorch/vision/pull/3496#pullrequestreview-619539316

Reply:
Also, we'll need docs now :)

Reply:
yep instance method makes sense

there's already a quite detailed doctstring in `find_classes()` function, shall I move it to this method instead?

Reply:
I would link to `find_classes()` and  write something like

> Same as :func:`find_classes`. This method can be overridden to only consider a subset of classes, or to adapt to a different dataset directory structure.